core/server/master/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -392,4 +392,11 @@ void workerRegisterStream(
    * @param workerId the workerId of target worker
    */
   void removeDecommissionedWorker(long workerId) throws NotFoundException;
+
```

Comment:
this add worker id is used to keep the block id map across masters consistent so that when failover happens, the new elected primary master can still generate unique worker ids.